.env.example

```
@@ -0,0 +1 @@
+API_URL=http://<IPv4>:4000/graphql
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)